civitai-downloader/src/cli.py

```
@@ -0,0 +1,168 @@
+"""Command-line interface for CivitAI downloader."""
+
+import click
+import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)